src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/IndicatorMeta.cs

```diff
@@ -1,14 +1,41 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Converters;
 using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
 
 namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
 
-public class IndicatorMeta
+public class IndicatorMeta : ICreatedUpdatedTimestamps<DateTimeOffset, DateTimeOffset?>
 {
-    public required string Identifier { get; set; }
+    public int Id { get; set; }
```

Comment:
`Guid`

Reply:
**Discussed:**

Only using `Guids` for publicly available IDs. Using `int` for strictly internally available IDs as it is more performant.